nbconvert/nbconvertapp.py

```diff
@@ -238,21 +238,20 @@ def initialize(self, argv=None):
 
 
     def init_syspath(self):
-        """
-        Add the cwd to the sys.path ($PYTHONPATH)
-        """
+        """Add the cwd to the sys.path ($PYTHONPATH)"""
```

Comment:
One thing is that reformatting for style screw up git blame, but I thins it's ok if there is just small change like that for this time.